ipn/ipnlocal/local.go

```diff
@@ -3081,6 +3081,8 @@ func (b *LocalBackend) authReconfig() {
 //
 // The versionOS is a Tailscale-style version ("iOS", "macOS") and not
 // a runtime.GOOS.
+//
+// XXX doesn't seem to use nm arg?
```

Comment:
no XXX in submitted code

Reply:
You're just the person I wanted to ask! Does it make sense to remove this arg?

Reply:
dropping the param sgtm, though do it in its own commit